content/en/docs/tasks/tools/_index.md

```diff
@@ -2,5 +2,40 @@
 title: "Install Tools"
 description: Set up Kubernetes tools on your computer.
 weight: 10
+no_list: true
 ---
 
+## kubectl
+
+The Kubernetes command-line tool, `kubectl`, allows you to run commands against
+Kubernetes clusters. You can use kubectl to deploy applications, inspect and manage
+cluster resources, and view logs.
+
+[Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/) explains how to
+download and install `kubectl` and set it up for accessing your cluster.
```

Comment:
I like this sentence ✨ , but it's better practice to avoid starting sentences with links. Might we rephrase this? Feel free to reword to your satisfaction.
```suggestion
See [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/) for information about how to
download and install `kubectl` and set it up for accessing your cluster.
```